.circleci/config.yml

```diff
@@ -131,14 +131,14 @@ jobs:
             --build-arg BUILD_TAG=${CIRCLE_SHA1} \
             --build-arg APP_BRANCH=${CIRCLE_BRANCH} \
             --build-arg BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) \
-            -t $AWS_ECR_ACCOUNT_URL/crime-apps/laa-crown-court-proceeding-dev-ecr:${CIRCLE_SHA1} .
```

Comment:
I still see the team name in the name space as crime-apps. Was the namespace changed?

Reply:
cloud-platform-environments/namespaces/live.cloud-platform.service.justice.gov.uk/laa-crown-court-proceeding-dev/resources/variables.tf